cluster/libvirt-coreos/util.sh

```diff
@@ -279,6 +302,7 @@ function kube-up {
   export KUBE_SERVER="http://192.168.10.1:8080"
   export CONTEXT="libvirt-coreos"
   create-kubeconfig
+  create-kubelet-kubeconfig "http://${MASTER_IP}:8080/" "${POOL_PATH}/kubernetes/kubeconfig/kubelet.kubeconfig"
```

Comment:
Not as far as I know. I suppose we could just stick it in the same dir as the certs...